dist/ractivef-amd.js

```diff
@@ -1,143 +1,68 @@
 define(['Ractive'], function (Ractive, RactiveF) {
 /* jshint ignore:start */
 RactiveF = {
+	VERSION: '0.0.22',
 	components: {},
 	templates: {},
-	widgets: [],
-	initInstance: function (container) {
-
-		// Have we mixed in extensions to all instances yet?
-		if (!Ractive.prototype.findAllChildComponents) {
-			_.mixin(Ractive.prototype, RactiveF.mixins);
-		}
-
-		var instance = new Ractive({
-			el: container,
-			template: Ractive.parse(container.innerHTML),
-			components: RactiveF.components,
-			onrender: function () {
-				this.el.classList.remove('hide');
-				this.el.classList.add('initialize');
-			}
-		});
-
-		instance.on('*.*', RactiveF.genericEventHandler);
-
-		instance.set('dataModel', '{{dataModel}}');
-
-		return instance;
+	getInstance: function () {
+		return Ractive.extend(RactiveF);
+	},
+	forge: function (options) {
+		options = options || {};
+		var Instance = RactiveF.getInstance();
+		return new Instance(options);
 	},
 
-	genericEventHandler: function (origin) {
-
-		// list of events below copied from Ractive source code v0.7.1
-		// Filtering out ractive lifecycle events to not pollute log output.
-		var reservedEventNames =
-				/^(?:change|complete|reset|teardown|update|construct|config|init|render|unrender|detach|insert)$/;
-
-		if (!reservedEventNames.test(this.event.name)) {
-			console.log('Event', this.event.name);
-			console.log('Event handler arguments', origin);
-
-			var eventName = 'events.' + origin.get('uid');
-			if (!this.get(eventName)) {
-				this.set(eventName, []);
-			}
-			this.push(eventName, this.event.name);
-		}
-
+	/*
+	 * When working with nested components we only want to find child
+	 * components, not all decendants.
+	 * @param name
+	 */
+	findAllChildComponents: function (name) {
+		return _.filter(this.findAllComponents(name), function (component) {
```

Comment:
I know that the dependency on Lodash was here before, but since we refactor should we be explicit in ```define([``` ?